vulnfeeds/cves/versions.go

```diff
@@ -613,10 +394,45 @@ func Commit(u string) (string, error) {
 
 	// TODO(apollock): add support for resolving a GitHub PR to a commit hash
 
+	// Support for resolving a Github tag to a commit hash
+	// example: https://github.com/redis/redis/releases/tag/6.2.17
+	if parsedURL.Host == "github.com" {
+		possibleCommitHash, err := resolveGitTag(parsedURL, u, gitSHA1Regex)
+		if possibleCommitHash != "" && err == nil {
+			return possibleCommitHash, nil
+		}
+	}
 	// If we get to here, we've encountered an unsupported URL.
 	return "", fmt.Errorf("Commit(): unsupported URL: %s", u)
 }
 
+func resolveGitTag(parsedURL *url.URL, u string, gitSHA1Regex *regexp.Regexp) (string, error) {
+	directory, tag := path.Split(parsedURL.Path)
+	normalizedTag, err := git.NormalizeVersion(tag)
```

Comment:
Do we need to continue if this fails? Since we are matching against normalized tags further down anyway if this fails we can probably just exit here.